### ISSUE TITLE
Custom Instructions

### ISSUE THREAD

**Issue by fishcharlie** [github/CopilotForXcode#28]
It'd be great if Copilot for Xcode supported [Custom Instructions](https://docs.github.com/en/copilot/customizing-copilot/adding-custom-instructions-for-github-copilot).

**Comment by devm33** [comment]
Hi @fishcharlie, thanks for the feedback. Definitely on the roadmap!

**Comment by kapoorlakshya** [comment]
Hey @devm33, any idea on the timeline?

**Comment by testforstephen** [comment]
@kapoorlakshya For Chat feature, we have enabled support for custom instructions defined in `.github/copilot-instructions.md` within your workspace.

**Comment by jordikitto** [comment]
Hi @testforstephen, any plans to support multiple instruction files for specific directories via `.instructions.md`, as specified [here](https://docs.github.com/en/enterprise-cloud@latest/copilot/how-tos/configure-custom-instructions/add-repository-instructions?tool=webui#using-one-or-more-instructionsmd-files-1)

We have a multi-platform project, so just one `instructions.md` file is not enough, we need to be able to specify which sub-projects it should be applied to. Ideally something like in the link would be great.

**Comment by testforstephen** [comment]
@jordikitto yes, we have plan to support `*.instructions.md` as well, stay tuned.
